docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -0,0 +1,402 @@
+# Machine Learning Training
+
+Machine Learning Training allows to construct, control and train a machine learning model in Tizen devices.
```

Comment:
I think we need to give little bit more details (briefly) about Machine Learning Training.
Something like..

Machine Learning Training API allows you to train your model in Tizen Devices.

The main features of the Machine Learning Training API include:

Construct deep neural network Easily
You can construct DNN model by Configuration File or writing code through Machine Learning Training API.
....
Train with your own data
Machine Learning API also allows you to train the model with your own data as a File I/O and defining data generator code.
....
Evaluate the model during train
You can validate and test your model during train process easily by define the data set.
...

Reply:
Updated